draft-irtf-cfrg-aead-limits.md

```diff
@@ -533,11 +543,24 @@ v * 2^64 + (2l * (v + q))^2 <= (p / u) * 2^128
 
 # Security Considerations {#sec-considerations}
 
-Many of the formulae in this document depend on simplifying assumptions that are
-not universally applicable.  When using this document to set limits, it is
-necessary to validate all these assumptions for the setting in which the limits
-might apply.  In most cases, the goal is to use assumptions that result in
-setting a more conservative limit, but this is not always the case.
+Many of the formulae in this document depend on simplifying assumptions,
+from differing models, that are not universally applicable. When using this
+document to set limits, it is necessary to validate all these assumptions
+for the setting in which the limits might apply. In most cases, the goal is
+to use assumptions that result in setting a more conservative limit, but this
+is not always the case. As an example of one such simplification, this document
+defines v as the total number of failed decryption queries, whereas in certain
+models, such as {{GCMProofs}}, it is defined as the total number of queries.
+
+The CA and IL values defined in this document are upper bounds based on existing
+cryptographic research. Future analysis may introduce tighter bounds. Applications
+SHOULD NOT assume these bounds are rigid, and SHOULD accommodate changes.
```

Comment:
Good point.  Maybe clarify with something like: "In particular, in two-party communication, one participant cannot regard apparent overuse of a key by other participants as being in error, when it could be that the other participant has better information about bounds."

Reply:
Agree that's an important point for (not) hard-coding limits. And I like the perspective that your communication partner might be cryptographically better informed ;-)

Reply:
Yeah, great suggestion!